guide/02-api-overview/overview24.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>mview.draw() -&gt; m.content.draw()</p><p> </p><p>Duplicate row for <code>m.center</code> in property table</p><p> </p><p>I can see both class urls for <code>mview.time_mode</code> and <code>mview.time_slider</code> notes, might reviewnb issue...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2053/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>